packages/ember-htmlbars/lib/keywords/mut.js

```diff
@@ -63,11 +63,32 @@ let MutStream = ProxyStream.extend({
   // my-child.js
   export default Component.extend({
     click() {
-      this.attrs.childClickCount.update(this.attrs.childClickCount.value + 1);
+      this.get('childClickCount').update(this.get('childClickCount').value + 1);
     }
   });
   ```
 
+  Additionally, the `mut` helper can be combined with the `action` helper to more
```

Comment:
Yea I had kind of wondered about that. I'd be happy to remove that if you'd like. I felt like it was good to try to provide some context when you might utilize the `mut` helper in this way.